test/http-multipart.js

```diff
@@ -42,6 +47,22 @@ describe('buildRequest - openapi 2.0', () => {
       expect(itemEntries.length).toEqual(2)
       expect(itemEntries[0]).toEqual('person1')
       expect(itemEntries[1]).toEqual('person2')
+      // set of non-multi collectionFormat cases below
```

Comment:
I'd suggest to return to my original comment https://github.com/swagger-api/swagger-js/pull/1527#discussion_r423757494 about how we approach building tests here. We're just adding stuff into the test and making it bigger and more complex. IMHO building tests in this manner will not scale for future and maintainability.

Reply:
done. separated and categorized 'expects"